pkg/inspecttypes/dockercompat/dockercompat.go

```diff
@@ -232,15 +233,25 @@ func ContainerFromNative(n *native.Container) (*Container, error) {
 			}
 			c.AppArmorProfile = p.ApparmorProfile
 		}
+		c.Mounts = mountsFromNative(sp.Mounts)
+		for _, mount := range c.Mounts {
+			if mount.Destination == "/etc/resolv.conf" {
+				c.ResolvConfPath = mount.Source
+			}
+			if mount.Destination == "/etc/hostname" {
+				c.HostnamePath = mount.Source
+			}
```

Comment:
nit: we can use `else if` at line 241 to save an unnecessary `if`.

Reply:
Fixed.